recipe/meta.yaml

```
@@ -528,9 +525,8 @@ outputs:
         - {{ pin_subpackage('libopenvino-paddle-frontend', exact=True) }}
         - {{ pin_subpackage('libopenvino-pytorch-frontend', exact=True) }}
         - {{ pin_subpackage('libopenvino-tensorflow-frontend', exact=True) }}
-        - numpy <2.2.0
+        - numpy <2.3.0
```

```suggestion
        - numpy <2.4.0
```

https://github.com/openvinotoolkit/openvino/blob/2025.4.0/src/bindings/python/requirements.txt#L1